tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -390,12 +390,6 @@ def prepare_ir(argv: argparse.Namespace):
     # TODO: remove this workaround once new TensorFlow frontend supports non-frozen formats: checkpoint, MetaGraph, and SavedModel
     # Now it converts all TensorFlow formats to the frozen .pb format in case new TensorFlow frontend
     is_tf, _, _, _, _ = deduce_legacy_frontend_by_namespace(argv)
```

Comment:
@popovaan, please move this comment along with the code.
Please do it this in the separate PR.